var/spack/repos/spack_repo/builtin/packages/exawind/package.py

```diff
@@ -35,6 +35,7 @@ class Exawind(CMakePackage, CudaPackage, ROCmPackage):
 
     depends_on("c", type="build")
     depends_on("cxx", type="build")
+    depends_on("fortran", type="build")
```

Comment:
Confirmed we need that, since we unconditionally depend on `openfast`: https://github.com/Exawind/exawind-driver/blob/421ca84cc371755ad1bcefe8a509159b9a2f234c/CMakeLists.txt#L45